src/sdc11073/mdib/clientmdib.py

```diff
@@ -408,10 +408,22 @@ def _waitUntilInitialized(self, log_prefix):
         if showsuccesslog:
             self._logger.info('{}: _waitUntilInitialized took {} seconds', log_prefix, delay)
 
+    @staticmethod
+    def _copy_report_node(report_node: etree_._Element) -> etree_._Element:
+        """
+        Copy and keep complete namespace. See https://github.com/Draegerwerk/sdc11073/issues/191
+
+        :param report_node: report node to be copied
+        :return: new report node
+        """
+        new_parent = etree_.Element(report_node.tag, nsmap=report_node.nsmap)
```

Comment:
This element did not even exit before.
It is really not a good idea to generate new data here.
Keep it as close to the original as possible.

Reply:
done